Dockerfile

```
@@ -12,12 +12,12 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     ffmpeg \
     && rm -rf /var/lib/apt/lists/*
 
-RUN pip install markitdown
+RUN pip install markitdown fastapi uvicorn
 
 # Default USERID and GROUPID
 ARG USERID=10000
 ARG GROUPID=10000
 
 USER $USERID:$GROUPID
 
-ENTRYPOINT [ "markitdown" ]
+ENTRYPOINT ["uvicorn", "src.markitdown.api:app", "--host", "0.0.0.0", "--port", "8000"]
```

This doesn't work: `src/markitdown/api.py` doesn't exists, so the API will not start.

You must copy the `src` directory:

```suggestion
COPY src /src

ENTRYPOINT ["uvicorn", "src.markitdown.api:app", "--host", "0.0.0.0", "--port", "8000"]
```

Note that in `.dockerignore` that everything is ignored, so the `COPY` won't work.  
Modify the `.dockerignore` so the  `src/`-folder gets included:

```ignore
*
!/src
```